server/resources/User.py

```diff
@@ -44,3 +60,36 @@ def delete(cls, user_id: int):
 
         user.delete_from_db()
         return {"message": USER_DELETED}, 200
+
+class UserLogin(Resource):
+    @classmethod
+    def post(cls):
+        user_json = request.get_json()
+        user_data = user_schema.load(user_json)
+
+        user = UserModel.find_by_email(user_data.email)
+
+        if user and safe_str_cmp(user_data.password, user.password):
+            access_token = create_access_token(identity=user.id, fresh=True)
+            refresh_token = create_refresh_token(user.id)
+            return {"access_token": access_token, "refresh_token": refresh_token}, 200
+        
+        return{"message": INVALID_CREDENTIALS}, 401
+
+class UserLogout(Resource):
+    @classmethod
+    @jwt_required
+    def post(cls):
+        jti = get_raw_jwt()["jti"] #jwt id for jwt
```

Comment:
is "jti" a convention?

Reply:
I believe it is, https://flask-jwt-extended.readthedocs.io/en/latest/blacklist_and_token_revoking.html